django_stomp/execution.py

```diff
@@ -80,6 +83,7 @@ def _callback(payload: Payload) -> None:
                     raise e
                 finally:
                     local_threading.request_id = None
+                    db.close_old_connections()
```

Comment:
One really cool thing is that it [practically guarantees](https://github.com/django/django/blob/main/django/db/__init__.py#L38) that it won't generate an error when closing the connection